src/Form/Extension/CollectionTypeExtension.php

```diff
@@ -24,6 +24,8 @@ public function configureOptions(OptionsResolver $resolver): void
             [
                 'allow_drag_and_drop' => true,
                 'add_button_label' => 'forms.buttons.addItem',
+                'min' => 0,
+                'max' => null,
```

Comment:
I would call this `maximum_allowed_items`

Reply:
fixed [dd884eb](https://github.com/sumocoders/FrameworkCoreBundle/pull/138/commits/dd884eb960aaceed002f0d33c351ffce59253eb4)